### PR TITLE
Update Querly and Brakeman docs

### PR DESCRIPTION
**Pull request by wata727**
`gems` option is now available in Querly and Brakeman.

### CODE REVIEW

File: docs/tools/ruby/querly.md [hunk 1]
```diff
@@ -22,7 +22,30 @@ Visit the Querly project page for more information about writing rules:
...
+
+## Analyzing view templates
+
+Querly has a mechanism called [preprocessor](https://github.com/soutaro/querly/blob/master/manual/configuration.md#preprocessor) for analyzing templates which contain Ruby code.
+Sider finds the following gems in `Gemfile.lock` and installs them automatically for backward compatibility:
+
+- [slim](https://github.com/slim-template/slim)
+- [haml](https://github.com/haml/haml)
+
+These gems will not be installed when the `gems` option is specified. We encourage you to explicitly specify gems in the [`gems` option](../../getting-started/custom-configuration.md#gems-option) in `sideci.yml`.
```
Comment (akito19) [inline docs/tools/ruby/querly.md]: Is the first sentence conflicted with the second one? Slim and Haml will not be installed when we don't use `gems` option?

Comment (wata727) [reply docs/tools/ruby/querly.md]: Umm, the first one means "These gems are not installed **automatically**". I will change the sentence.

Comment (akito19) [reply docs/tools/ruby/querly.md]: I got it 👍

**Comment by ivanha09** [comment]
LGTM!

**Pull request merged**